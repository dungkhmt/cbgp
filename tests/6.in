30 30
30 30
27 3
14 1
28 2
17 1
19 1
5 0
19 10
17 5
25 0
8 16
27 1
8 22
2 4
18 29
25 26
21 1
24 11
27 16
9 16
7 4
6 0
28 29
12 11
21 13
28 23
25 11
18 20
4 0
15 28
5 1
1
5 1 21 1 0 0 14 8 1 1 13 3 4 2 2 2 15 5 14 4 26 3 10 4 12 4 27 1 29 9 29 1 16 4 13 1 35 0 24 3 38 0 25 1 13 5 28 3 11 5 8 4 12 8 18 4 31 0 33 0 