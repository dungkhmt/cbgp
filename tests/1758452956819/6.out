(27, 3), (14, 1), (28, 2), (17, 1), (19, 1), (5, 0), (19, 10), (17, 5), (25, 0), (8, 16), (27, 1), (8, 22), (2, 4), (18, 29), (25, 26), (21, 1), (24, 11), (27, 16), (9, 16), (7, 4), (6, 0), (28, 29), (12, 11), (21, 13), (28, 23), (25, 11), (18, 20), (4, 0), (15, 28), (5, 1), 
numberIntersection=0 
minDistance=4.160251471689219 
sumDistance=73674.69643870083 
minAngle=0.04477292588348192 
sumAngle=21.19803280442001 
minEdgeLength=4.242640687119285 
sumEdgeLength=853.7446359111127 

0.0000000000 4.1602514717 73674.6964387008 0.0447729259 21.1980328044 4.2426406871 853.7446359111
0: (113, 114),
1: (115, 16),
2: (120, 117),
3: (0, 5),
4: (117, 120),
5: (110, 109),
6: (105, 120),
7: (112, 120),
8: (0, 10),
9: (5, 15),
10: (0, 0),
11: (5, 111),
12: (8, 115),
13: (111, 9),
14: (110, 20),
15: (115, 6),
16: (5, 10),
17: (115, 109),
18: (72, 1),
19: (5, 0),
20: (80, 0),
21: (115, 11),
22: (0, 15),
23: (95, 8),
24: (5, 106),
25: (0, 120),
26: (0, 99),
27: (5, 5),
28: (120, 0),
29: (81, 5),
113 114 115 16 120 117 0 5 117 120 110 109 105 120 112 120 0 10 5 15 0 0 5 111 8 115 111 9 110 20 115 6 5 10 115 109 72 1 5 0 80 0 115 11 0 15 95 8 5 106 0 120 0 99 5 5 120 0 81 5 
0: (113, 114), 1: (115, 16), 2: (120, 117), 3: (0, 5), 4: (117, 120), 5: (110, 109), 6: (105, 120), 7: (112, 120), 8: (0, 10), 9: (5, 15), 10: (0, 0), 11: (5, 111), 12: (8, 115), 13: (111, 9), 14: (110, 20), 15: (115, 6), 16: (5, 10), 17: (115, 109), 18: (72, 1), 19: (5, 0), 20: (80, 0), 21: (115, 11), 22: (0, 15), 23: (95, 8), 24: (5, 106), 25: (0, 120), 26: (0, 99), 27: (5, 5), 28: (120, 0), 29: (81, 5), 
numberIntersection=0 minAngle=0.04477292588348192 sumAngle=21.198032804405052 minEdgeLength=4.242640687119285 sumEdgeLength=853.7446359111163 minDistance=4.160251471689219 sumDistance=73674.69643870031 
Time taken: 7.033 s
